### DISCUSSION TITLE
Process completed with exit code 1 - Error on Pull Request

### DISCUSSION THREAD

**Discussion by GeBon22** [neuefische/questions#337] [Web] [answered]
This error only occurs when trying to set a pull request. The test catches while doing "Run npm run prettier" and points at my HeaderAnimation.json file -> which I added following the instructions given by LottieFiles step by step. 
As I am planning to use the .json Animation File I cannot simply cut it out of the code. I tried to comment the section out of the code using */. That did not work. 

![grafik](https://user-images.githubusercontent.com/114496506/204511734-3b06b94c-6808-4327-be0b-f6c6cc5e03b4.png)

**Comment by GeBon22** [comment]
![grafik](https://user-images.githubusercontent.com/114496506/204513319-33385087-0564-493e-81f1-9969e15d2eb5.png)

FYI this is the component that uses the .json animation file.

**Comment by actyralf** [comment, marked as answer]
Please try to run `npm run prettier:write` - this should format all your files according to the settings in `.prettierrc`.
Run `npm run prettier` afterwards to check if the test passes...

**Reply by GeBon22** [reply]
Thats it, cheers.
